### PR TITLE
DHCP Relay Stress Test v1

### PR DESCRIPTION
**Pull request by yyynini**
### Description of PR

Summary:
The DHCP Relay Stress Test Plan is to evaluate the performance of the DHCP relay service under maximum load conditions. This test is to ensure the service can handle under a high traffic condition and verify the CPU utilisation.

test description: https://github.com/sonic-net/sonic-mgmt/issues/3614
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

### CODE REVIEW

File: docs/testplan/DHCP-relay-stress-test.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## Test structure 
+===============
+
+### Configuration
+-------------------
+
+1) def setup_copp_policy(): define a CoPP policy on the test device, and limit DHCP traffic to a maximum rate 600.
+2) def client_send_discover(): simulate client sending DHCPDISCOVER message by broadcast. Duration: 120s. e.g. PTF will send 1,000 packets per second.
+```
+duration = 120
```
Comment (kevinskwang) [inline docs/testplan/DHCP-relay-stress-test.md:49]: I think we don't need paste code here

File: docs/testplan/DHCP-relay-stress-test.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+| loganalyzer    | check no error keywords related to DHCP and CoPP |
+
+## Test structure 
+===============
+
+### Configuration
+-------------------
+
+1) def setup_copp_policy(): define a CoPP policy on the test device, and limit DHCP traffic to a maximum rate 600.
+2) def client_send_discover(): simulate client sending DHCPDISCOVER message by broadcast. Duration: 120s. e.g. PTF will send 1,000 packets per second.
```
Comment (kevinskwang) [inline docs/testplan/DHCP-relay-stress-test.md:47]: Can we send some more like 10K pkt/s?

File: docs/testplan/DHCP-relay-stress-test.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Test structure 
+===============
+
+### Configuration
+-------------------
+
+1) def setup_copp_policy(): define a CoPP policy on the test device, and limit DHCP traffic to a maximum rate 600.
+2) def client_send_discover(): simulate client sending DHCPDISCOVER message from different source MAC by broadcast. Duration: 120s. e.g. PTF will send 10,000 packets per second.
+3) def client_send_request(): simulate client sending DHCPREQUEST message from different source MAC or through different interface.
```
Comment (yaqiangz) [inline docs/testplan/DHCP-relay-stress-test.md:48]: In common dhcp_relay scenario, we don't have DHCP server for DUT. But I found `DHCP server sends message`, is the function `server_send_xxx` is missed here?

File: docs/testplan/DHCP-relay-stress-test.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+### Scope
+---------
+
+CoPP is to limit the rate of traffic sent to CPU, and then generating packets above this maximum rate, the test is to ensure the DHCP relay service can handle the maximum load without failure.
+
+### Scale / Performance
+-------------------
+1. Verfy DHCP Container Status.
+2. CPU should remain within acceptable range, indicating the DUT is not overwhelmed.
+3. Check whether the function is still normal.
```
Comment (yaqiangz) [inline docs/testplan/DHCP-relay-stress-test.md:26]: Could you please share the detail of checking dhcp_relay functionality? My suggestion is to verify whether DHCP packets are relayed immediately or with little delay

**Review by yaqiangz — APPROVED** [review]
LGTM

**Review by kevinskwang — APPROVED** [review]

**Pull request merged**